AzureDataLake/Scripts/Process.usql

```diff
@@ -1,22 +1,23 @@
-﻿USE DATABASE [DecisionServiceDB];
+﻿SET @@FeaturePreviews = "DataPartitionedOutput:on";
+
+USE DATABASE [DecisionServiceDB];
 
 REFERENCE ASSEMBLY DecisionServiceDB.[DecisionServiceExtractor];
 
 DECLARE @_appId string = "";    //your appId
 DECLARE @_account string = "";    // your account name
 
-DECLARE @_year string = "2019";  //your date: year
-DECLARE @_month string = "08";     //your date: month
-DECLARE @_day string = "12";       //your date: day
+DECLARE @_start DateTime = new DateTime(2019, 10, 1);  //your inclusive start date
+DECLARE @_end DateTime = new DateTime(2019, 10, 12);    //your non-inclusive end date
 
-DECLARE @OutputFolder string = "/daily/";      //your output folder
-DECLARE @OutputSlimLogs bool = false;     //true if you want to have slime version of logs (all metadata without features)
+DECLARE @OutputFolder string = "/" + @_appId + "_daily/";      //your output folder
```

Comment:
can we do:
`DECLARE @OutputFolder string = "/daily/" + @_appId + "/";`

Reply:
sure.